### PR TITLE
feat: Update vektor and systemspec

### PR DESCRIPTION
**Pull request by arbourd**
Updates vektor to v0.6.1

- Updates Middleware to be wrappable
- Updates handlers for new function signatures
- Replaces `vk.R` with `vk.RepondJSON` or `vk.RepondBytes`

### CODE REVIEW

File: sat/sat/handler.go [hunk 1]
```diff
@@ -62,6 +62,6 @@ func (s *Sat) handler(exec *executor.Executor) vk.HandlerFunc {
 			ctx.RespHeaders.Set(headerKey, headerValue)
 		}
 
-		return resp.Output, nil
+		return vk.RespondBytes(ctx.Context, w, resp.Output, http.StatusOK)
```
Comment (arbourd) [inline sat/sat/handler.go:65]: I've made an assumption that a successful response also yields a `200`.

**Comment by javorszky** [comment]
Before looking at code, the middleware is already wrappable! Hol' up! 😅

**Review by javorszky — APPROVED** [review]
:shipit:

**Review by jagger27 — APPROVED** [review]

**Comment by arbourd** [comment]
Tested this with Compose and SE2.

**Pull request merged**